src/Common/Auth/OpenIDConnect/Repositories/ScopeRepository.php

```diff
@@ -298,11 +418,13 @@ public function systemScopes(): array
             "system/Consent.read",
```

Comment:
would it make sense to comment out all the other system scopes like we do for patient (or just remove them since sounds like never plan to implement them)